src/mygrad/tensor_base.py

```diff
@@ -626,6 +648,9 @@ def backward(self, grad=None):
         >>> x.grad # df/dx = df/dw * dw/dx
         array(6.)
         """
+        if not _track.TRACK_GRAPH:
+            return
```

Comment:
Is it even valid to call `.backward()` in `no_autodiff` mode?

Reply:
My thoughts are that I want users to be able to basically just throw their training code under the `no_autodiff` context and have it be valid for performing inference. From this perspective, I think making `.backward()` simply be a no-op.